models/classes/LtiLaunchData.php

```diff
@@ -435,6 +452,62 @@ public function getOauthKey()
         return $this->getVariable(self::OAUTH_CONSUMER_KEY);
     }
 
+    /**
+     * @throws LtiVariableMissingException
+     */
+    public function getLtiForUserId(): ?string
```

Comment:
What is the difference in scenarios between `null` value and throwing`LtiVariableMissingException`, which for me may look quite similar? ( applies not only here, but also for other newly introduced variables bellow )

Reply:
my first think was that exception thrown only when we don't have requested key in storage, but after check i see there is no difference. i will fix it right now